acceptance/tests/sameness/sameness_test.go

```diff
@@ -428,21 +428,30 @@ func TestFailover_Connect(t *testing.T) {
 				"../fixtures/cases/sameness/static-server/dc3")
 
 			// Create static client deployments.
+			staticClientDefaultKustomizeDir := "../fixtures/cases/sameness/static-client/default-partition"
+			staticClientAP1KustomizeDir := "../fixtures/cases/sameness/static-client/ap1-partition"
+
+			// If transparent proxy is enabled create clients without explicity upstreams
```

Comment:
```suggestion
			// If transparent proxy is enabled create clients without explicit upstreams
```

Reply:
I see Gollum wrote that comment "explicity hobitsys"